src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <section id="court-order-poa">
+    <header>
+      <h2>{{sectionNumber}} Court Order and Plan of Arrangement</h2>
+      <div class="py-4">
+        If this filing is pursuant to a court order, enter the court order number. If this
+        filing is pursuant to a plan of arrangement, enter the court order number and select
+        Plan of Arrangement.
+      </div>
+    </header>
+    <div :class="{ 'invalid-section': invalidCourtOrder }">
+      <CourtOrderPoaShared
+        id="court-order"
+        :autoValidation="getAppValidate"
+        :hasDraftPlanOfArrangement="hasDraftPlanOfArrangement"
+        :invalidSection="invalidCourtOrder"
+        @emitCourtNumber="setFileNumber($event)"
+        @emitPoa="setHasPlanOfArrangement($event)"
+        @emitValid="setValidCourtOrder($event)"
+      />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { CourtOrderPoa as CourtOrderPoaShared } from '@bcrs-shared-components/court-order-poa'
+import { ActionBindingIF, FlagsReviewCertifyIF } from '@/interfaces/'
+
+@Component({
+  components: {
+    CourtOrderPoaShared
+  }
+})
+export default class CourtOrderPoa extends Vue {
+  /** Prop to provide section number. */
+  @Prop({ default: '' }) readonly sectionNumber: string
+
+  /** Store getters */
+  @Getter getFlagsReviewCertify!: FlagsReviewCertifyIF
+  @Getter getAppValidate!: boolean
+  @Getter getHasPlanOfArrangement!: boolean
+
+  /** Global actions */
+  @Action setValidCourtOrder!: ActionBindingIF
+  @Action setHasPlanOfArrangement!: ActionBindingIF
+  @Action invalidCourtOrder!: ActionBindingIF
+
+  /** LocalProperties */
+  private courtOrderNumber: string = null
+  private planOfArrangement: boolean = false
```

Comment:
I think you don't need these any more.

Does VS Code show you grayed-out variables/properties when they're unused? Like this:

![image](https://user-images.githubusercontent.com/10002889/169399369-504dc3fd-76fd-4721-b4a5-6857190dcc6e.png)

If not, you're probably missing a VS Code extension. RC me for details.

Reply:
Is `setValidCourtOder` not being used on line 19 at `@emitValid="setValidCourtOrder($event)"`

Reply:
This was an example from a different project. Sorry for the confusion.

Reply:
Right on you are right they are not needed.